### PR TITLE
Add BONK to scope SDK

### PR DESCRIPTION
**Pull request by oeble**

### CODE REVIEW

File: packages/scope-sdk/src/Scope.ts [hunk 1]
```diff
@@ -58,6 +58,22 @@ export const ScopePairs = [
...
   'kUXDUSDCOrca/USD',
   'JITOSOL/SOL',
+  'SOLEma/USD',
+  'ETHEma/USD',
+  'SRMEma/USD',
+  'BTCEma/USD',
+  'RAYEma/USD',
+  'FTTEma/USD',
+  'MSOLEma/USD',
+  'BNBEma/USD',
```
Comment (peroxy) [inline packages/scope-sdk/src/Scope.ts:68]: How come you also added a bunch of other pairs here? Is this related to BONK?

**Review by peroxy — APPROVED** [review]

File: packages/scope-sdk/src/Scope.ts (hunk 1 above)
Comment (oeble) [reply packages/scope-sdk/src/Scope.ts:68]: Not related, but the index matter IIRC. Those are in scope even if unused in the SDK, I added them so I have the right index.

File: packages/scope-sdk/src/Scope.ts [hunk 2]
```diff
@@ -200,6 +217,7 @@ export class Scope {
     { id: 49, pair: 'UXP/USD', name: 'UXP', price: new Decimal(0) },
     { id: 50, pair: 'kUXDUSDCOrca/USD', name: 'kUXDUSDCOrca', price: new Decimal(0) },
     { id: 51, pair: 'JITOSOL/SOL', name: 'JITOSOL', price: new Decimal(0), nonUsdPairId: 0 },
+    { id: 52, pair: 'BONK/USD', name: 'BONK', price: new Decimal(0) },
```
Comment (y2kappa) [inline packages/scope-sdk/src/Scope.ts:220]: the ID doesn't seem right here

Comment (oeble) [inline packages/scope-sdk/src/Scope.ts:220]:
```suggestion
    { id: 68, pair: 'BONK/USD', name: 'BONK', price: new Decimal(0) },
```

File: packages/scope-sdk/src/Scope.ts [hunk 3]
```diff
@@ -114,6 +130,21 @@ export const SupportedTokens = [
   'UXP',
   'kUXDUSDCOrca',
   'JITOSOL',
+  'SOLEma',
+  'ETHEma',
+  'SRMEma',
```
Comment (alittlezz) [inline packages/scope-sdk/src/Scope.ts:135]: No BTC here?

Comment (y2kappa) [reply packages/scope-sdk/src/Scope.ts:135]: there is no good BTC in solana yet..

Comment (alittlezz) [reply packages/scope-sdk/src/Scope.ts:135]: then remove BTC from above

Comment (y2kappa) [reply packages/scope-sdk/src/Scope.ts:134]: removed

**Review by alittlezz — APPROVED** [review]

**Review by silviutroscot — APPROVED** [review]
